src/files/manifest.js

```diff
@@ -29,6 +29,7 @@ export const createManifest = async (context, config, queue, options) => {
         display: options.display,
         theme_color: options.themeColor,
         background_color: options.backgroundColor,
+        gcm_sender_id: (options.gcmSenderId? options.gcmSenderId : undefined),
```

Comment:
ternary here is redundant IMO

Reply:
not sure to understand what do you mean?

Reply:
Why do need to set `gcm_sender_id` to undefined in case of a  falsy value of `options.gcmSenderId` ?

The better solution is to have default value of `options.gcmSenderId` as `undefined` instead of `''` if that is a concern.

Reply:
ok got it